### PR TITLE
Added tutorial for trying to stay small and to check the help menu

### LINKED ISSUE #2773: Add tutorial on 3rd swimming cycle that points to the help menu
so that players know that it is a thing if they are still lost after completing the tutorial

### PR DESCRIPTION
**Pull request by hhyyrylainen**
also added the mouse to zoom control in the help menu, also refactored a common class out of nth editor entry tutorial

**Related Issues**

closes  #2773

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author.

### CODE REVIEW

File: src/tutorial/microbe_stage/CheckTheHelpMenu.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+﻿namespace Tutorial
+{
+    using System;
+    using Newtonsoft.Json;
+
+    public class CheckTheHelpMenu : TutorialPhase
+    {
+        public const string TUTORIAL_NAME = "CheckTheHelpMenu";
+
+        private const int TriggersOnNthSwimmingSession = 3;
```
Comment (adQuid) [inline src/tutorial/microbe_stage/CheckTheHelpMenu.cs:10]: Might want to rename this TriggersOnNthEditorSession since that's what this class counts

Comment (hhyyrylainen) [reply src/tutorial/microbe_stage/CheckTheHelpMenu.cs:10]: But this is the tutorial for the microbe stage? So I renamed this variable when copy-pasting.

File: src/tutorial/microbe_editor/StaySmallTutorial.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    public class StaySmallTutorial : EditorEntryCountingTutorial
+    {
+        public StaySmallTutorial()
+        {
+            CanTrigger = false;
+        }
+
+        public override string ClosedByName => "StaySmallTutorial";
+
+        protected override int TriggersOnNthEditorSession => 3;
```
Comment (adQuid) [inline src/tutorial/microbe_editor/StaySmallTutorial.cs:12]: By round three, isn't it a little late to tell the player to not add anything?

Comment (hhyyrylainen) [reply src/tutorial/microbe_editor/StaySmallTutorial.cs:12]: Key prompt showing doesn't work for the mouse wheel, and also keypad keys don't work, would need for MICROBE_STAGE_HELP_MESSAGE_1, fix that once implemented

maybe the try to stay small tutorial should not activate if the player has 2 or less organelles

Add new editor help menu explaining the main controls of the editor

MICROBE_STAGE_HELP_MENU_AND_ZOOM should use the actual binding for [thrive:input]g_zoom_out[/thrive:input]
 though maybe the ordering of the auto-evo prediction and this tutorial can be changed. I'll add this on my TODO list to open an issue.

Comment (hhyyrylainen) [reply src/tutorial/microbe_editor/StaySmallTutorial.cs:12]: Deleted my comment text...
anyway I said that the order of the auto-evo prediction and this tutorial could be swapped, but I don't want to front load any more tutorials to the first editor session.

**Comment by adQuid** [comment]
To be honest, I would recommend just keeping the check help tutorial, and move the strategy comments to the help text. It's a well-established fact that players don't read game text until they are ready, and I think a pop-up saying, "if you are willing to read if you get better, click the question mark" is the most we can realistically expect players to respond to. This game already has a word-heavy opening.

**Comment by hhyyrylainen** [comment]
I worry that (as I didn't have space to mention this) the player doesn't realize to check the editor help menu separately.

File: src/tutorial/microbe_editor/StaySmallTutorial.cs (hunk 2 above)
Comment (FuJa0815) [reply src/tutorial/microbe_editor/StaySmallTutorial.cs:12]: The player can't really mess up in the first session, I think, unless they place an early game nucleus or something stupid, so thats fine for me

File: locale/en.po [hunk 3]
```diff
@@ -4283,6 +4300,18 @@ msgstr ""
 "To leave the colony entirely you can click on your own cell or press the "
 "unbind all key."
 
+#: ../src/tutorial/microbe_stage/MicrobeTutorialGUI.tscn:421
+msgid "MICROBE_STAGE_HELP_MENU_AND_ZOOM"
+msgstr ""
+"If you are still struggling with game mechanics, check the help menu. You "
+"can open it with the question mark button in the bottom left.\n"
```
Comment (FuJa0815) [inline locale/en.po:4307]:
```suggestion
"can open it with the question mark button in the bottom left or by pressing [thrive:input]help[/thrive:input].\n"
```
Not tested but if this works, why not add that?

**Review by FuJa0815 — APPROVED** [review]
`tgtm`

In all seriousness, looks good. Not sure if it's possible to show the help-shortcut in the tutorial. If this works, I would add that, if not, looks good.

File: locale/en.po (hunk 3 above)
Comment (hhyyrylainen) [reply locale/en.po:4307]: It doesn't actually work. I think we have an open issue.

Comment (hhyyrylainen) [reply locale/en.po:4307]: Found it: https://github.com/Revolutionary-Games/Thrive/issues/2462

**Pull request merged**